app/models/patient.rb

```diff
@@ -1,13 +1,18 @@
 # Object representing core patient information and demographic data.
+
 class Patient
   include Mongoid::Document
   include Mongoid::Timestamps
   include Mongoid::Enum
   include Mongoid::History::Trackable
   include Mongoid::Userstamp
   include StatusHelper
+  include Urgency
+  include Callable
+  include Notetakeable
+  include Searchable
+  include AttributeDisplayable
 
```

Comment:
I found the [default docs](http://api.rubyonrails.org/classes/ActiveSupport/Concern.html) a bit lacking in use depth, but [this nice writeup was helpful](http://vaidehijoshi.github.io/blog/2015/10/13/stop-worrying-and-start-being-concerned-activesupport-concerns/)